docs(opentelemetry): migrate troubleshooting.md from github (#1923)
SumoLogic/sumologic-documentation: docs/send-data/choose-collector-source.md

The Sumo Logic Distribution for [OpenTelemetry Collector](https://github.com/open-telemetry/opentelemetry-collector) is built with the [opentelemetry-collector-builder](https://github.com/open-telemetry/opentelemetry-collector-builder) and provides a single unified agent to send Logs, Metrics, Traces, and Metadata for Observability to Sumo Logic.

> Refer to [OpenTelemetry Collector document](/docs/send-data/opentelemetry-collector/) for more information about the Sumo Logic Distribution for OpenTelemetry Collector.

Reviewer: ```suggestion
Refer to [Sumo Logic Distribution for OpenTelemetry Collector](/docs/send-data/opentelemetry-collector/) documentation for more information.
```

> It's supported on *Linux*, *MacOS*, *Windows* and *Kubernetes* environments and can use any of the following Sources:

Reviewer: rm italics

```suggestion
It's supported on Linux, macOS, Windows, and Kubernetes environments and can use any of the following Sources:
```
